frontend/src/metabase/visualizations/components/Visualization/Visualization.tsx

```diff
@@ -267,7 +425,20 @@ class Visualization extends PureComponent {
       : [];
   }
 
-  visualizationIsClickable = clicked => {
+  getNormalizedSizes = () => {
+    const { width, height } = this.props;
+
+    return {
+      width: width ?? 0,
+      height: height ?? 0,
+    };
+  };
```

Comment:
Extracted to prevent adding additional logic to the `render` method that increases its compexity (eslint check)